src/classes/customer.js

```diff
@@ -0,0 +1,11 @@
+class Customer {
```

Comment:
And so does this! :)

Smart thinking ahead and adding the userName and password when you're in the flow and know you'll be using them in the future. I didn't do this and, seeing yours, wish that I had.